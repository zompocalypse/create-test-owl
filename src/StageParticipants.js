import React from 'react';
import './StageParticipants.css'

export default function StageParticipants(props) {
  return (
    <div className="onStageParticipants">
        <img src={props.avatar} alt={props.name}/>
        <div className="stageDetails">
          <span className="contact-on-stage-details-text">
          {props.name}
          </span>
          <div className="mute">mute</div>
          <div className="search">search</div>
        </div>
    </div>
  )
}