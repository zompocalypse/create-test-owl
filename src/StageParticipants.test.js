import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import StageParticipants from './StageParticipants';
import store from './store';

describe( 'StageParticipant component' , () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<StageParticipants />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders UI as expected', () => {
    const tree = renderer
      .create(<StageParticipants
        name={store.participants[0].name}
        avatar={store.participants[0].avatar}
      />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

})