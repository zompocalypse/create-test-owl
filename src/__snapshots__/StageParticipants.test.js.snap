// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StageParticipant component renders UI as expected 1`] = `
<div
  className="onStageParticipants"
>
  <img
    alt="Koren Templeton"
    src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
  />
  <div
    className="stageDetails"
  >
    <span
      className="contact-on-stage-details-text"
    >
      Koren Templeton
    </span>
    <div
      className="mute"
    >
      mute
    </div>
    <div
      className="search"
    >
      search
    </div>
  </div>
</div>
`;
